kinder/pkg/cri/containerd/alterhelper.go

```diff
@@ -86,6 +92,15 @@ func PullImages(bc *bits.BuildContext, images []string, targetPath string) error
 	return nil
 }
 
+// PreLoadInitImages preload images required by kubeadm-init into the containerd runtime that exists inside a kind(er) node
+func PreLoadInitImages(bc *bits.BuildContext, srcFolder string) error {
+	// NB. this code is an extract from "sigs.k8s.io/kind/pkg/build/node"
+	return bc.RunInContainer(
+		"bash", "-c",
+		`find `+srcFolder+` -name *.tar -print0 | xargs -0 -n 1 -P $(nproc) ctr --namespace=k8s.io images import --all-platforms --no-unpack`,
```

Comment:
my understanding is that this preload does not work and i think you verified this during our zoom?
did you make it work somehow in this PR?

from my tests, after the container is started images are not there and they end up being pulled from the internet, which caused kubeadm CI to pull some extra gigabytes from the web.